js/template/ProfileSettings/ProfileSettingsViewProfile.js

```diff
@@ -81,6 +82,15 @@ export default class ProfileSettingsViewProfile extends ProfileSettingsView {
         );
         emailInput.loadInContext(this.root);
 
+        const receiveNotificationsInput = new LabelGroup(
+            'Receive Notifications',
+            new CheckboxInputTemplate({isEnabled: Data.shared.valueForKey(Key.userReceiveNotifications) == 1}),
+            {
+                tooltip: 'Choose whether or not you want to receive notifications about updates to your content.',
```

Comment:
I believe `isHorizontalStyle: true` should also be set otherwise tooltip won't show alongside the checkbox